registry/registry.go

```diff
@@ -87,15 +87,15 @@ func (r *RouteRegistry) Register(uri route.Uri, endpoint *route.Endpoint) {
 		return
 	}
 
-	endpointAdded := r.register(uri, endpoint)
+	result := r.register(uri, endpoint)
```

Comment:
(I know this change was made by me) We should probably put this back and defer that to the round of clean-up we plan to do afterwards.